robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -122,9 +128,9 @@ def enter_value_in_field(self,**kwargs):
                 self.salesforce._populate_field(locator, value)   
 
     @capture_screenshot_on_error
-    def select_object_group_field(self,object_group,field):
-        """Select the specified field under specified object group 
-           to add the field to gift entry form and verify field is added"""
+    def perform_action_on_object_field(self,action,object_group,field):
+        """If action is 'select' then selects the specified field under specified object group to add the field to gift entry form and verify field is added
+        If action is 'unselect'then Unselects the specified field under specified object group to remove the field from gift entry template"""
         locator=npsp_lex_locators["gift_entry"]["form_object_dropdown"].format(object_group)
         self.salesforce._jsclick(locator)
         element=self.selenium.get_webelement(locator)
```

Comment:
I assume we are storing some locator here. Are we using this anywhere in this method?

Reply:
yes in line 137, I am using it to get "aria-expanded" attribute value